source/mainnet/smart-contracts/tutorials/wCCD/wCCD-front-end-set-up.rst

```diff
@@ -4,6 +4,354 @@
 Setting up the front-end
 ========================
 
-Coming soon.
+Non-tech users might find it inconvenient to interact via the Concordium node with smart contracts
+and some users may choose not to host their own Concordium node in the future.
+
+You can lower the bar for entry by coding an appealing front-end that provides additional information
+to your potential customers. Web front-ends are a familiar sight nowadays, but keep in mind
+that downloading a crypto wallet as a browser extension and its behavior or safe usage might be new
+for people using your front-end. Providing comprehensive explanations and step-by-step guides on your website on topics
+related to the browser wallet is important for a good user experience. The browser wallet
+connects via ``HTTPS`` to a server that is connected to a Concordium node. This setup alleviates the
+need for the user to host their own Concordium node.
+
+.. note::
+
+    Before you start with part 3 of this tutorial, make sure you have:
+
+    - access to the `Chrome <https://www.google.com/chrome/>`_ web browser on your laptop or computer
+
+    - `git <https://git-scm.com/>`_ installed to be able to clone a repository from `GitHub <https://github.com/>`_
+
+    - `node.js <https://nodejs.org/en/download/>`_ and `yarn <https://yarnpkg.com/getting-started/install>`_ installed
+
+    ``Node.js`` is a JavaScript runtime environment and ``yarn`` is a package manager
+    used in combination with ``node.js``. You can download `node.js <https://nodejs.org/en/download/>`_  and
+    `yarn <https://yarnpkg.com/getting-started/install>`_  on their official websites.
+
+Concordium browser wallet
+-------------------------
+
+A browser wallet is a piece of code that can be added as an extension to supported browsers such as ``Chrome``.
```

Comment:
Shouldn't `Chrome` be a link?

Reply:
Chrome is linked 3 sentences above (when it is first mentioned). I then just highlighted the name to show it is a name instead of linking it every time to an external side.